cypress.json

```diff
@@ -1,5 +1,9 @@
 {
   "baseUrl": "http://localhost:3000",
+  "integrationFolder": "cypress/specs",
+  "pluginsFile": "cypress/plugins.js",
+  "supportFile": "cypress/support.js",
+  "screenshotsFolder": "cypress/debug",
   "fixturesFolder": false,
-  "pluginsFile": false
+  "video": false
```

Comment:
Recording a video is time-consuming and a bit over the top for us. I think the debug screenshots are enough.

Note that I also rename a couple of folders, and I remove the unnecessary `plugins` and `support` folders by pointing directly at files at the root of the `cypress` folder.